src/media/css/detail/more-info.styl

```diff
@@ -5,10 +5,13 @@
 @import '../lib';
 
 .app-more-metadata {
+        margin: 0 -12px;
```

Comment:
This should be indented by 4 spaces so it line up with the `dl` below. Please add a blank line after also.